src/App.js

```diff
@@ -20,7 +26,7 @@ function App() {
           <Home />
         </Route>
         <Route path="/use-coins">
-          <UseCoins />
+          <UseCoins spendCoins={spendCoins} lastEarning={lastEarning} />
```

Comment:
The total of coins can be spend, otherwise this would just be a distribution, no?